README.md

```diff
@@ -65,83 +65,68 @@ For more information and instructions, see the Kubernetes blog post, [Kubernetes
 
 ##### 4.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
 
-
 <br/>
 <br/>
 <br/>
 
-
 ## Installation
 
-### Install the operator
+### Installing the driver with GitHub
```

Comment:
note the formatting that I have in https://github.com/IBM/ibm-block-csi-driver/blob/documentation/docs/csi_ug_install_operator_github.md. I don't think you're indenting properly in order to keep the substeps

Reply:
AFAIK, the README doesn't really have indentations, just sub-titles..